File: python/prism_sdk/multimodal_benchmark_compilation_integrity_inference.py
"""Benchcompiler P32 multimodal inference benchmark-compilation integrity feature."""
from .benchmark_compilation_integrity_support import BenchmarkCard7,BenchmarkCompileRequest4,BenchmarkCompilationIntegrityError,manifest,compile
FEATURE_ID="AFA-benchcompiler-P32-F02";CONTRACT_VERSION="benchcompiler-multimodal_benchmark_compilation_integrity_inference/1.0"
def multimodal_benchmark_compilation_integrity_inference_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="multimodal",mode="inference")
def compile_multimodal_benchmark_compilation_integrity_inference(request:BenchmarkCompileRequest4)->BenchmarkCard7:return compile(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="multimodal",mode="inference")
__all__=["FEATURE_ID","CONTRACT_VERSION","multimodal_benchmark_compilation_integrity_inference_manifest","compile_multimodal_benchmark_compilation_integrity_inference"]

File: crates/benchcompiler/src/multimodal_benchmark_compilation_integrity_inference.rs
//! Benchcompiler P32 multimodal inference benchmark-compilation integrity feature.
use super::benchmark_compilation_integrity_support::{manifest,compile,BenchmarkCard7,BenchmarkCompileRequest4,BenchmarkCompilationIntegrityError};
pub const FEATURE_ID:&str="AFA-benchcompiler-P32-F02";pub const CONTRACT_VERSION:&str="benchcompiler-multimodal_benchmark_compilation_integrity_inference/1.0";
pub fn multimodal_benchmark_compilation_integrity_inference_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"multimodal","inference")}
pub fn compile_multimodal_benchmark_compilation_integrity_inference(request:&BenchmarkCompileRequest4)->Result<BenchmarkCard7,BenchmarkCompilationIntegrityError>{compile(request,FEATURE_ID,CONTRACT_VERSION,"multimodal","inference")}
